crates/bevy_tasks/src/task_pool.rs

```diff
@@ -309,6 +336,42 @@ mod tests {
         assert_eq!(count.load(Ordering::Relaxed), 100);
     }
 
+    #[test]
+    fn test_spawn_pollable() {
+        let transform_fn = |i: usize| i * 3;
+
+        let pool = TaskPool::new();
+        let nums: Vec<usize> = (1..10).into_iter().collect();
+        let pollables: Vec<PollableTask<usize>> = nums
+            .clone()
+            .into_iter()
```

Comment:
```suggestion
            .iter()
            .copied()
```
avoids the allocation